app/models/manageiq/providers/redhat/infra_manager/refresh/parse/strategies/api4.rb

```diff
@@ -1,4 +1,41 @@
 module ManageIQ::Providers::Redhat::InfraManager::Refresh::Parse::Strategies
   class Api4 < ManageIQ::Providers::Redhat::InfraManager::Refresh::Parse::Parser
+    def self.cluster_inv_to_hashes(inv)
+      result = []
+      result_uids = {}
```

Comment:
The ids used by oVirt may be UUIDs, and may not. I think a better name for this is `result_ids`.